src/core/format-utils.js

```diff
@@ -0,0 +1,43 @@
+import BigNumber from 'bignumber.js';
+
+export const DOCK_TOKEN_UNIT = 1000000;
+
+export function formatCurrency(value, currency = 'USD') {
+  var formatter = new Intl.NumberFormat('en-US', {
```

Comment:
can we format numbers using the user's locale instead of hard-coding to en-US?

Reply:
In react native we need to fetch the current locale from the system, a regular web approach will not work here. I think it should be handled later, when we translate the locales.

Reply:
Can you create a ticket and start tracking changes you're aware of that we'd need to do for that? It's come up in recent customer calls.

Reply:
Yes https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871